src/app/subcategory/components/createSubcategoryForm/createSubcategoryForm.tsx

```diff
@@ -145,7 +146,11 @@ export const CreateSubcategoryForm = ({
                 maxLength={MAX_FIELD_LENGTH}
               />
             </Form.Control>
-            {errorMessage && <p className="error-text">{errorMessage}</p>}
+            {errorMessage && errorMessage === USER_TOKEN_ERROR ? (
```

Comment:
this part of code is used in lot of files. Should it be component then which returns GoToLoginPage or <p>?
smth like
<ErrorHandler error={errorMessage} />